ntp-client/src/Network/NTP/Client.hs

```diff
@@ -65,6 +56,54 @@ data NtpClientSettings = NtpClientSettings
     }
 
 data NtpClient = NtpClient
+    { ntpGetStatus        :: IO NtpStatus -- Use STM here ?
+      -- ^ Query the current status (non-blocking).
+    , ntpActNow           :: IO ()
+      -- ^ Bypass all internal threadDelays and trigger a new ntp query.
```

Comment:
Do we need this? Isn't `ntpForcecheck` enough?

Reply:
Yes maybe its enought..I Just wanted to keep the existing thing and add the new new Idea so it does not get lost. Also there is no real implementation of ntpForceCheck yet.

Reply:
There is, but it's not explicit (by updating the `TVar`).

Reply:
No need to keep both api's at the same time, but if you want to keep it for the time being of working on this PR that's ok.